src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working");
```

Comment:
The message in `doWork` method should be more informative and indicate the type of machine that is working. For example, 'Bulldozer started its work.'